csfieldguide/chapters/content/en/coding-compression/sections/image-compression-using-jpeg.md

```diff
@@ -183,7 +183,7 @@ Notice how the images look very similar, even though the second one has many zer
 
 Try this out yourself:
 
-{interactive slug="jpeg-compression" type="whole-page" text="JPEG interactive"}
+{comment interactive slug="jpeg-compression" type="whole-page" text="JPEG interactive"}
```

Comment:
I'm guessing this is an interactive without a set thumbnail, I didn't notice it before (regarding earlier comments). In that case my solution is just to add one (have `jpeg-compression/img/thumbnail.png` exist - empty file or whatever you wish), the other is to comment out the tests like what's been done in the Complexity and Tractability PR. Both are temporary, and Jack's working on the proper resolution this week

Reply:
actually, I have no idea why this diff is showing like this. When I go through this files history it even shows that I removed the `comment` part of this line, so I am baffled as to why this is here. Have removed it again and this interactive appears to be in working order, thumbnail and all.